Automattic/wp-calypso
# Adds eCommerce plan constants.

#### Changes proposed in this Pull Request

* Adds constants and necessary getter methods for eCommerce plan.

#### Testing instructions
* Make sure tests pass.
* Make sure you can import new plan constants `PLAN_ECOMMERCE`, and type `TYPE_ECOMMERCE` from `lib/plans/constants.js`.
